tools/salmon-quant-tool.cwl

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env cwl-runner
+class: CommandLineTool
+id: salmon-quant-tool
+label: salmon-quant-tool
+cwlVersion: v1.0
+
+requirements:
+  - class: DockerRequirement
```

Comment:
Update syntax here.  

Its completely fine for us to use the `- class....` formatting.  I just don't know if you wanted uniform syntax across all workflow and tools.